style/base.css

```diff
@@ -14,7 +14,7 @@
   justify-content: space-between;
   margin-top: 2px;
   font-family: var(--jp-code-font-family, monospace);
-  font-size: 80%;
+  font-size: calc(0.8*14px);
```

Comment:
Why not just raise it from 80% to a higher number to compensate; that way if someone has a bigger/smaller font set, its relative and not absolute

Reply:
Done.